site.yml

```diff
@@ -157,6 +157,14 @@
       tags: ['ceph', 'ceph-osd']
   environment: "{{ env_vars|default({}) }}"
 
+- name: ceph update
+  hosts: ceph_monitors:ceph_osds:controller:compute
```

Comment:
This is to allow for the `ceph.conf` file to be updated on all the nodes where its present, hence ceph monitors+osds along with controllers and computes. The handlers in `ceph-common`, which this role is replacing in a scaleout situation, also restarted ceph services on all these nodes, when `site.yml` was run with the `ceph` tag.